packages/plugin-lightningcss/src/types.ts

```diff
@@ -0,0 +1,38 @@
+import type { CustomAtRules, TransformOptions } from 'lightningcss';
+
+type Implementation = typeof import('lightningcss');
```

Comment:
A loose type is better here, such as `unknown`. Otherwise the user can not pass a different version of lightningcss (the type will be mismatch).